src/command/expire/expire.c

```diff
@@ -502,9 +513,12 @@ removeExpiredArchive(InfoBackup *infoBackup)
                                 // Remove the entire directory if all archive is expired
                                 if (removeArchive)
                                 {
+                                  // execute the real expiration and deletion
+                                  // only if the dry-run mode is disabled
+                                  if ( ! cfgOptionBool(cfgOptDryRun) )
                                     storagePathRemoveP(
-                                        storageRepoWrite(), strNewFmt(STORAGE_REPO_ARCHIVE "/%s/%s", strPtr(archiveId),
-                                        strPtr(walPath)), .recurse = true);
+                                                       storageRepoWrite(), strNewFmt(STORAGE_REPO_ARCHIVE "/%s/%s", strPtr(archiveId),
+                                                                                     strPtr(walPath)), .recurse = true);
```

Comment:
Since this IF statement spans more than one line, wrap it in curly braces {} (See CODING.md). Please apply spacing standards and curly braces to your other code changes where needed.

Reply:
done